ui/lib/kubernetes/addon/components/page/overview.js

```diff
@@ -0,0 +1,35 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+export default class OverviewPageComponent extends Component {
+  @service router;
+
+  @tracked roles = [];
+  @tracked selectedRole = null;
+  @tracked roleOptions = this.roles.map((role) => {
+    return { name: role.name, id: role.name };
+  });
+
+  constructor() {
+    super(...arguments);
+    this.roles = this.args.model.roles.toArray();
```

Comment:
You should be able to remove the tracked `roles` property and access this directly in the template via `@model.roles`.